.github/workflows/build.yaml

```
@@ -44,7 +44,7 @@ jobs:
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
   upload-conda:
-    needs: [cpp-build, python-build]
+    needs: [cpp-build, python-build, java-build]
```

Revert, since no `java-build` job exists anymore.
```suggestion
    needs: [cpp-build, python-build]
```